pkg/errors/errors.go

```diff
@@ -37,3 +40,11 @@ func newNoRouteToHost(errString string) *KNError {
 func newNoKubeConfig(errString string) *KNError {
 	return NewKNError("no kubeconfig has been provided, please use a valid configuration to connect to the cluster")
 }
+
+func firstCharToUpper(s string) string {
```

Comment:
+1

Actually, I would use this function, too, before emitting an error message to the console, so that error messages that start with a lowercase letter (golang convention) will be upcased. There should be a single exit-point for errors to do the massaging of the error message to print. But that's of course another story ;-)